composer.json

```diff
@@ -17,18 +17,20 @@
         }
     ],
     "require": {
-        "doctrine/orm": "^2.5.3",
         "php": "^7.2",
+        "doctrine/orm": "^2.5.3",
         "sulu/sulu": "^2.0.3 || ^2.1@dev",
         "symfony/config": "^4.3 || ^5.0",
         "symfony/console": "^4.3 || ^5.0",
         "symfony/dependency-injection": "^4.3 || ^5.0",
         "symfony/event-dispatcher": "^4.3 || ^5.0",
         "symfony/event-dispatcher-contracts": "^1.0 || ^2.0",
         "symfony/form": "^4.3 || ^5.0",
-        "symfony/validator": "^4.3 || ^5.0",
+        "symfony/mime": "^4.4 || ^5.1",
+        "symfony/mailer": "^4.4 || ^5.1",
```

Comment:
We should not increase the dependency tree and as sulu ships currently with swiftmailer. So currently this should be just `dev` dependencies and if somebody want to use the `Mailer` implementation he need to install the symfony mailer component.